### PR TITLE
feat: translate web-components from map to views

### PR DESCRIPTION
**Pull request by flagrede**
## Description

translate web-components from map to views

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-marketplace/src/components/atoms/ExampleProjectLink.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Link as RouterLink } from 'react-router-dom';
```
Comment (blushi) [inline web-marketplace/src/components/atoms/ExampleProjectLink.tsx:1]: why do we need this? does not seem to be used

File: web-components/src/components/modal/add-wallet-modal/AddWalletModalConnect.tsx [hunk 2]
```diff
@@ -1,18 +1,25 @@
 import { RegenModalProps } from '..';
```
[the remark this answers is not in the document]
Comment (flagrede) [reply web-components/src/components/modal/add-wallet-modal/AddWalletModalConnect.tsx:1]: This one, we can remove it, I think.

File: web-components/src/components/modal/add-wallet-modal/AddWalletModalRemove.tsx [hunk 3]
```diff
@@ -5,24 +5,23 @@ import {
   Props as CancelButtonFooterProps,
```
[the remark this answers is not in the document]
Comment (flagrede) [reply web-components/src/components/modal/add-wallet-modal/AddWalletModalRemove.tsx:5]: This one too.

File: web-components/src/components/modal/add-wallet-modal/AddWalletModalSwitch.tsx [hunk 4]
```diff
@@ -1,10 +1,15 @@
 import { RegenModalProps } from '..';
```
[the remark this answers is not in the document]
Comment (flagrede) [reply web-components/src/components/modal/add-wallet-modal/AddWalletModalSwitch.tsx:1]: This one too.

File: web-marketplace/src/components/atoms/ExampleProjectLink.tsx (hunk 1 above)
Comment (flagrede) [reply web-marketplace/src/components/atoms/ExampleProjectLink.tsx:1]: It's something I extracted from a [component](https://github.com/regen-network/regen-web/blob/0502b306deff636c49237d70e0d5059499d67619/web-components/src/components/section/OnBoardingSection.tsx#L90-L97). But this part was never used. Should I remove this component or add some comment to indicate it's related to the `OnBoardingSection` component ?

File: web-components/src/components/modal/add-wallet-modal/AddWalletModalRemove.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (flagrede) [reply web-components/src/components/modal/add-wallet-modal/AddWalletModalRemove.tsx]: Removed.

File: web-components/src/components/modal/add-wallet-modal/AddWalletModalConnect.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (flagrede) [reply web-components/src/components/modal/add-wallet-modal/AddWalletModalConnect.tsx]: Removed.

File: web-marketplace/src/components/atoms/ExampleProjectLink.tsx (hunk 1 above)
Comment (blushi) [reply web-marketplace/src/components/atoms/ExampleProjectLink.tsx:1]: oh I see let's just drop it then

File: web-components/src/components/modal/SwitchWalletWarningModal/SwitchWalletWarningModal.tsx [hunk 5]
```diff
@@ -1,32 +1,32 @@
 import { SadBeeIcon } from '../../../components/icons/SadBeeIcon';
```
[the remark this answers is not in the document]
Comment (blushi) [reply web-components/src/components/modal/SwitchWalletWarningModal/SwitchWalletWarningModal.tsx:1]: oh right, I was looking only for the full string `setSwitchWalletModal` while it's actually used more often as `setSwitchWalletModalAtom`

**Review by blushi — APPROVED** [review]
LGTM

**Pull request merged**